src/handler/patch.ts

```diff
@@ -0,0 +1,64 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { addHttpHeaders } from '../util/httpHeaders';
+import {
+  archiveOldCreateCurrentRecord,
+  getBySystemNumberAndCreatedTimestamp,
+} from '../services/database';
+import { validateUpdateVinRequest, validateVins } from '../validators/patch';
+import { PatchRequestRecords } from '../processors/processPatchVinRequest';
+import { TechrecordGet } from '../models/post';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Patch Technical Record Called');
```

Comment:
Do you think this would be more appropriate to be changed to amend vin called, something along those lines?